### ISSUE TITLE
I would like to have a strategy: Karmada will prioritize pods to cluster A when scheduling pods, and will only schedule pods to cluster B when cluster A is insufficient (pending), how can this process be automated?

### ISSUE THREAD

**Issue by Schwarao** [karmada-io/karmada#5997]
I would like to have a strategy: Karmada will prioritize pods to cluster A when scheduling pods, and will only schedule pods to cluster B when cluster A is insufficient (pending), how can this process be automated?

**Comment by RainbowMango** [comment]
Hi @Schwarao I think the [affinity groups](https://karmada.io/docs/userguide/scheduling/resource-propagating/#multiple-cluster-affinity-groups) might help.

**Comment by Schwarao** [comment]
> 您的要求是这样的吗？https://github.com/karmada-io/karmada/pull/5512/files
> 
> 我们正在针对这个问题制定最佳实践，如果你也有这个要求，那就太好了，我们可以邀请你一起讨论~

yes，This is indeed the requirement, but I only need to complete the pod resource type here

**Comment by Schwarao** [comment]
> Pod 资源要简单得多。您可以先尝试现有解决方案（[亲和组](https://karmada.io/docs/userguide/scheduling/resource-propagating/#multiple-cluster-affinity-groups)）。
> 
> 如果您对此解决方案有任何不满意的地方，请及时反馈。

Is it the content of the Multiple cluster affinity groups?

**Comment by Schwarao** [comment]
But it seems that it shouldn't work：
This is the policy I use：
![image](https://github.com/user-attachments/assets/19a6633e-1f1d-4683-b268-f25eca519ac2)

After creating a file here, it is still scheduled and in pending：
![image](https://github.com/user-attachments/assets/22b71ec0-ef32-4831-8223-f3d81ce0c68b)



> Pod resources are much simpler. You can try the existing solution([affinity groups](https://karmada.io/docs/userguide/scheduling/resource-propagating/#multiple-cluster-affinity-groups)) first.
> 
> If there is anything you don't satisfy with this solution, please give feedback in time.

**Comment by Schwarao** [comment]
@chaosi-zju 
I see, I want to add this step field:
replicaScheduling:
      replicaSchedulingType: Divided
      replicaDivisionPreference: Aggregated

#4694 
Does the content in the following document need to be added to the specification?
![image](https://github.com/user-attachments/assets/ae5b2452-f6b5-4f6c-bb93-9977284edd0e)

**Comment by Schwarao** [comment]
> > 是 Multiple cluster affinity groups （多集群关联组） 的内容吗？
> 
> 是的
> 
> > 在此处创建文件后，它仍处于 scheduled 状态且处于待处理状态：
> 
> 也许您没有指定或您的 IS 策略，那么它仍将安排到 First Group`replicaScheduling``replicaScheduling``StaticWeight`
> 
> > 以下文档中的内容是否需要添加到规范中？
> 
> 对不起，我不明白你的意思？您的意思是要改进我们的网站文档以防止用户错过这些信息吗？

Yes, there is no mention in the content of Multiple cluster affinity groups in the documentation that you need to specify replicaScheduling replicaScheduling StaticWeight, you see if you need to optimize
meantime，I have deployed multiple PropagationPolicies and now I want to create pods, how do I specify which PropagationPolicy to use in yaml
